packages/dcos-diagnostics/windows.build.ps1

```diff
@@ -0,0 +1,14 @@
+$SRC_DIR = "c:\gopath\src\github.com\dcos\dcos-diagnostics\"
+new-item -itemtype directory "c:\gopath\src\github.com\dcos"
+
+copy-item -recurse  -Path c:/pkg/src/dcos-diagnostics/ -Destination c:/gopath/src/github.com/dcos/
+
+Push-Location $SRC_DIR
+
+$env:GOOS = "windows"
+& go build .
```

Comment:
In Linux Makefile we also set Version and Commit, but we've never bump version so I'm fine with setting only commit.

https://github.com/dcos/dcos-diagnostics/blob/1389e1499b0790a9bdf041514fe173a395a4b7b8/Makefile#L3-L5